ckanext/qdes_schema/templates/package/snippets/validate_and_publish_metadata.html

```diff
@@ -122,18 +134,15 @@ <h2>Resources:</h2>
           {% endif %}
         </div>
         <div class="col-md-2">
-          {% if is_public and is_official_public %}
-            <button
-              data-module="publish-confirm-action"
-              data-module-content="Has the metadata record gone through the relevant business review / approval process to publish?"
-              data-module-content-cancel="The distribution can not be published without the relevant business review / approval process."
-              data-module-with-data="true"
-              class="btn btn-default btn-block publish"
-              type="submit" name="action" value="publish" disabled>Publish</button>
-          {% endif %}
+          <button
```

Comment:
Hi @awset 
I noticed that the `{% if is_public and is_official_public %}` condition around the Publish button has been removed in this PR - is this intentional?
Thanks,
Nathan.

Reply:
@salsa-nathan yes, it is intentional, I have make sure the AC those button need to be visible **but disabled** and that's enable/disable is happening in js. and on page load it is always disabled.